rust/lance-table/src/format/manifest.rs

```diff
@@ -117,6 +117,44 @@ fn compute_fragment_offsets(fragments: &[Fragment]) -> Vec<usize> {
         .collect()
 }
 
+#[derive(Default)]
+pub struct ManifestStats {
+    total_fragments: u64,
+    total_data_files: u64,
+    total_records: u64,
+    total_files_size: u64,
+    total_deletion_files: u64,
+    total_deletions: u64,
+}
+
+impl From<ManifestStats> for BTreeMap<String, String> {
+    fn from(val: ManifestStats) -> Self {
+        let mut stats_map = Self::new();
+        stats_map.insert(
+            "total-fragments".to_string(),
```

Comment:
kebab case does seem like an odd choice. I'm guessing this is the match the iceberg statistics?

Reply:
> kebab case does seem like an odd choice. I'm guessing this is the match the iceberg statistics?

The initial idea was inspired by Iceberg, and we utilized the summary for management functions. However, matching the naming convention wasn't intentional; I just didn't give it much thought. 

I've switched it to snake_case. Appreciate you catching that!